src/emysql_util.erl

```diff
@@ -52,6 +56,11 @@ result_type(R) -> emysql:result_type(R).
 as_dict(Res) -> emysql:as_dict(Res).
 as_json(Res) -> emysql:as_json(Res).
 as_proplist(Res) -> emysql:as_proplist(Res).
+
+-ifdef(maps_available).
+as_maps(Res) -> emysql:as_maps(Res).
+-endif.
+
 as_record(Res, RecName, Fields) -> emysql:as_record(Res, RecName, Fields).
 as_record(Res, RecName, Fields, Fun) -> emysql:as_record(Res, RecName, Fields, Fun).
```

Comment:
Line 65 is too long: as_record(Res, RecName, Fields, Fun) -> emysql:as_record(Res, RecName, Fields, Fun)..